elfi/examples/lotka_volterra.py

```diff
@@ -160,7 +162,12 @@ def get_model(n_obs=50, true_params=None, seed_obs=None, **kwargs):
     """
     logger = logging.getLogger()
     if true_params is None:
-        true_params = [1.0, 0.005, 0.6, 50, 100, 10.]
+        if observation_noise:
+            true_params = [1.0, 0.005, 0.6, 50, 100, 10.]
+        else:
+            true_params = [1.0, 0.005, 0.6, 50, 100, 0.]
+    elif not observation_noise and len(true_params) > 5 and true_params[5] != 0:
+        logger.warning("Option observation_noise=False. Ignoring sigma="+str(true_params[5])+".")
```

Comment:
the message says sigma is ignored, but that value is still used to generate `y_obs`. this means that the observed data in the returned model will have observation noise while the simulations do not. is this the intended outcome or should we set `true_params[5]=0` here?

Reply:
We should set `true_params[5]=0`